flatpak/README.md

```
@@ -142,7 +147,7 @@ Flathub has strict requirements for security and reproducibility:
    ```bash
    cd flatpak
    ./prepare_flathub_submission.sh
-   ```
+  ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)